lib/tasks/swagger_generate.rake

```diff
@@ -123,6 +123,67 @@ class SwaggerGenerator
     "#/definitions/#{collection_name}"
   end
 
+  def swagger_definition(klass_name)
+    {
+      "type"       => "object",
+      "properties" => swagger_definition_properties(klass_name),
+    }
+  end
+
+  def swagger_definition_properties(klass_name)
+    model = klass_name.constantize
```

Comment:
Does this need to `safe_constantize` instead?  There may be things that are not backed by models.

Reply:
I'd probably let it explode. And when it does, we'll need to figure out what to do with these. We'll probably let them to be defined manually?

Reply:
That's fine for now.